packages/halo/src/react/index.js

```diff
@@ -6,19 +6,19 @@ import React from 'react'
 import { withTheme } from '@pluralsight/ps-design-system-theme/react'
 import filterReactProps from '@pluralsight/ps-design-system-filter-react-props'
 
-import css, { BASE_CLASSNAME } from '../css'
+import css, { BASE_CLASSNAME, themeClasses } from '../css'
```

Comment:
While you're editing this file/line, you might as well add the /index.js filename to the pathing.